docs/fast_data/runtime_management.mdx

```diff
@@ -0,0 +1,147 @@
+---
+id: runtime_management
+title: Runtime Management
+sidebar_label: Runtime Management
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+
+
+# Runtime Management
+
+Welcome to Runtime Management - your tool for achieving better control over the Fast Data environment during runtime.
+This platform empowers you to dynamically manage the execution of Fast Data, offering functionalities to pause and resume microservices effortlessly.
+Dive into the world of streamlined runtime control and take charge of your Fast Data deployment with ease.
+
+## Overview
+
+Runtime Management is specifically designed to facilitate the runtime deployment of Fast Data.
+It provides a user-friendly web interface and robust backend functionalities to enhance your control over the Fast Data microservices environment.
+
+With Runtime Management, users gain the ability to govern potentially every aspect of the Fast Data runtime.
+Currently, this powerful mechanism is supported in the Single View Creator, offering more than just the suspension of Single View generation on MongoDB.
+
+### Key Features and Benefits
+
```

Comment:
Why should you use the Runtime Management?

Reply:
Sorry, I can't get what you mean. You're suggesting to change the name from Runtime Management to something else, or that the title of this pharagraph whould be the one you wrote?

Reply:
You're right, I'm sorry, I didn't explain what I meant here. I would add the answer to this question as the introduction of the paragraph. I would not start with the bullet points but I would add a simple sentence to introduce the benefits that you want to describe

Reply:
Ok, I refactored the info, removing scenarios that could create confusion and merged the ones that descirbe the same scenario.